games/devtest/mods/gltf/init.lua

```diff
@@ -0,0 +1,48 @@
+local function register_entity(name, textures)
+	minetest.register_entity("gltf:" .. name, {
+		initial_properties = {
+			visual = "mesh",
+			mesh = "gltf_" .. name .. ".gltf",
+			textures = textures,
+		},
+	})
+end
+
+-- These do not have texture coordinates; they simple render as black surfaces.
+register_entity("minimal_triangle", {})
+register_entity("triangle_with_vertex_stride", {})
+register_entity("triangle_without_indices", {})
+do
+	local cube_textures = {"no_texture.png"} -- TODO provide proper textures
```

Comment:
Please specify another texture from devtest. This way it's not obvious whether the correct texture is used/shown.

Reply:
Will do. I'll perhaps create a new texture for this purpose.